apps/utils/draft_pull_request.py

```diff
@@ -37,10 +93,27 @@
     required=False,
 )
 @click.option(
-    "--base-branch", type=str, default=None, help="Name of base branch (if not given, 'master' will be used)."
+    "--base-branch", "-b", type=str, default=None, help="Name of base branch (if not given, 'master' will be used)."
+)
+@click.option("--title", "-t", type=str, default=None, help="Title of the PR and the first commit.")
+@click.option(
+    "--category",
+    "-c",
+    type=click.Choice(list(PR_CATEGORIES.keys()), case_sensitive=False),
+    help=f"Category of the PR. Only works if --title is used (it will be added to the PR title). A corresponding emoji will be pre-fixed to the PR title.\n {description}",
 )
-@click.option("--title", type=str, default=None, help="Title of the PR and the first commit.")
-def cli(new_branch: Optional[str] = None, base_branch: Optional[str] = None, title: Optional[str] = None) -> None:
+@click.option(
+    "--scope",
+    "-s",
+    help="Scope of the PR. Only works if --title is used (it will be added to the PR title).\n\n\n**Examples**: 'demography' for data work on this field, 'etl.db' if working on specific modules, 'wizard', etc.",
```

Comment:
I don't think having a scope is very useful, the title itself is usually enough. But up to you.